web-local/src/routes/(misc)/deploy/+page.svelte

```diff
@@ -31,6 +32,21 @@
   onMount(() => {
     void deployer.loginOrDeploy();
   });
+
+  let upgradeHref = "";
+  $: if ($org && $metadata.data) {
+    let cloudUrl = $metadata.data.adminUrl.replace(
+      "admin.rilldata",
+      "ui.rilldata",
+    );
+    if (cloudUrl === "http://localhost:9090") {
+      cloudUrl = "http://localhost:3000";
+    }
```

Comment:
What's going on here? This seems like a hack. Can we solve the root problem?

Reply:
It s a similar hack to runtime on local. I can add a comment.